### DISCUSSION TITLE
sumo-bandplot problem

### DISCUSSION THREAD

**Discussion by chicatonita** [SMTG-UCL/sumo#228] [Q&A]
raceback (most recent call last):   File "/home/beh/.local/bin/sumo-bandplot", line 8, in <module>     sys.exit(main())   File "/home/beh/.local/lib/python3.8/site-packages/sumo/cli/bandplot.py", line 886, in main     bandplot(   File "/home/beh/.local/lib/python3.8/site-packages/sumo/cli/bandplot.py", line 298, in bandplot     bs = vr.get_band_structure(line_mode=True)   File "/home/beh/.local/lib/python3.8/site-packages/pymatgen/io/vasp/outputs.py", line 972, in get_band_structure     if "" in kpoint_file.labels: TypeError: argument of type 'NoneType' is not iterable

**Comment by ajjackson** [comment]
It's easier to read the traceback if the linebreaks are left in! You can format as a code block using triple backticks or the "code" button in the editor.

It looks like the code was expecting to find a list of kpoint labels, but internally found a `None` instead. Was the KPOINTS file generated using `sumo-kgen`, or something else?
